docs/jest-integration.md

```diff
@@ -87,6 +87,58 @@ which can be specified through the CLI client.
 }
 ```
 
+## Enabling TypeScript Jest tests
+
+Jest supports execution of tests written in other languages than JavaScript via
+dedicated extensions. Probably most prominent is the TypeScript support, which
+can be enabled via [`ts-jest`](https://kulshekhar.github.io/ts-jest/).
+
+Describing all available configuration options is out of scope for this
+documentation, and we assume you already set up your TypeScript project. The
+following section shows a minimal configuration to enable TypeScript support for
+Jest fuzz tests. Furthermore, an example project is available at
+[jest_typescript_integration](../examples/jest_typescript_integration).
+
+In addition to the configuration shown in the last section, `ts-jest` has to be
+added as dev-dependency to the project.
+
+```shell
+npm install --save-dev ts-jest
+```
+
+The Jazzer.js runner configuration also needs to reference `ts-jest`, most
+commonly by setting the `preset` property to `ts-jest`. Also make sure to
+actually test files with the `.fuzz.ts` extension.
+
+```typescript
+{
+  displayName: {
+    name: "Jazzer.js",
+    color: "cyan",
+  },
+  preset: "ts-jest",
+  runner: "@jazzer.js/jest-runner",
+  testEnvironment: "node",
+  testMatch: ["<rootDir>/*.fuzz.[jt]s"],
+},
+```
+
+**Note**: To provide accurate coverage reports for TypeScript fuzz tests, make
+sure to enable the generation of source maps in the TypeScript compiler options:
+
+```json
+{
+	"compilerOptions": {
+		"sourceMap": true
+	}
+}
+```
+
+These settings should be enough to start writing Jest fuzz tests in TypeScript.
+
+**Note**: Using custom hooks written in TypeScript is currently not supported,
```

Comment:
Do bug detectors work with Jest+TypeScript?

Reply:
No, as those are loaded directly by our internal code and aren't processed by `ts-jest`.

Reply:
Command injection works. But maybe because it hooks the builtin functions directly?

Reply:
I think I misunderstood your initial question. Bug detectors work, but they can not be developed in TypeScript themselves.